yarn.lock

```diff
@@ -13682,21 +13682,6 @@ chokidar-cli@2.1.0:
     lodash.throttle "^4.1.1"
     yargs "^13.3.0"
 
```

Comment:
If we bumped chokidar to 3.5.1, yarn lock should be updated too.

Reply:
Can you explain what you mean?
I see a change in yarn.lock as it should be.